deno/payloads/v10/channel.ts

```diff
@@ -344,6 +344,13 @@ export interface APIMessage {
 	author: APIUser;
 	/**
 	 * Contents of the message
+	 *
+	 * The **`MESSAGE_CONTENT`** privileged gateway intent is required for verified applications to receive a non-empty value from this field
+	 *
+	 * In Discord Developers Portal, you need to enable the toggle of this intent of your application in **Bot > Privileged Gateway Intents**.
+	 * You also need to specify the intent bit value  (**`1 << 15`**) if you are connecting to the gateway
+	 *
+	 * See https://support-dev.discord.com/hc/en-us/articles/4404772028055
```

Comment:
Bolding in-line code has no effect. Also removed a double space and added "In **the** Discord Developers Portal...":

```suggestion
	 *
	 * The `MESSAGE_CONTENT` privileged gateway intent is required for verified applications to receive a non-empty value from this field
	 *
	 * In the Discord Developers Portal, you need to enable the toggle of this intent of your application in **Bot > Privileged Gateway Intents**.
	 * You also need to specify the intent bit value (`1 << 15`) if you are connecting to the gateway
	 *
	 * See https://support-dev.discord.com/hc/en-us/articles/4404772028055
```